src/app/dashboard/dashboard.component.ts

```diff
@@ -1,8 +1,48 @@
 import { Component, OnInit } from '@angular/core';
+import { ApiService } from '../Services/api.service';
+import { HttpErrorResponse } from '@angular/common/http';
+import { BalanceDto } from '../balanceDto';
 
 @Component({
   selector: 'app-dashboard',
   templateUrl: './dashboard.component.html',
   styleUrls: ['./dashboard.component.css'],
 })
-export class DashboardComponent {}
+export class DashboardComponent implements OnInit {
+  constructor(private apiService: ApiService) {}
+
+  errorClockIn: number;
+  errorClockOut: number;
```

Comment:
Z kodu wynika, że do tego pola przypisujemy response.status, a status to status - nie error. Dlatego to pole powinno się nazywać względem tego co przechowuje np. statusClockIn i statusClockOut